packages/dapp/src/components/Widgets/WidgetEvents.tsx

```diff
@@ -127,10 +164,13 @@ export function WidgetEvents() {
       trackEvent({
         action: TrackingActions.OnRouteHighValueLoss,
         category: TrackingCategories.WidgetEvent,
-        label: 'click-highValueLossAccepted',
+        label: 'click_high_value_loss_accepted',
```

Comment:
Do we want to make these strings also typed somewhere?

Reply:
Hmm, not sure if that is necessary. But it´s a good question anyways. Wdyt @Addminus

Reply:
+1

Reply:
That does not feel right to just type all possible labels here, but I´d do it if you think it´s adding value.
I´d still love to hear @Addminus opinion as he beared with me on this whole topic.